site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,435 @@
+| `author_id`  | the ID of the sender of an event                                             | server    | string            |
+| `text`       | the payload that you can send within an event (for example message content) | client    | string            |
+| `recipients` | the posibble values are `"all"`(default) and `"agents"`                      | client    | string            |
+| `properties` | event [properties](#properties)                                              | client    | properties object |
+
+
+
+### Message
+
+> A sample **message**:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "custom_id": "12345-bhdsa",
+  "order": 1,
+  "type": "message",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "text": "hello there",
+  "recipients": "all",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+A **message** is the most common event type. It allows you to send textual content to other chat users.
+
+### Annotation
+
+> A sample **annotation**:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "custom_id": "12312.301231238591134",
+  "order": 1,
+  "type": "annotation",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "text": "Sample annotation",
+  "annotation_type": "rating",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+This event type adds an annotation to the last thread. Sending an annotation **cannot** start a new thread, even when there's no active thread in a chat. It always goes to the latest already existing thread.
+
+The `annotation_type` field defines the type of annotation. It **cannot be empty**. It's customizable, so you can have your own annotation types. We used the `rating` type in the example.
+
+
+
+### Filled form
+
+A **filled form** is an event containing data from a form, such as the one in the picture below.
+
+
+> A sample **filled form**:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "custom_id": "12312.301231238591134",
+  "order": 4,
+  "type": "filled_form",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "properties": {
+    // "Properties" object
+  }
+  "fields": [{
+      "type": "text",
+      "name": "name",
+      "label": "Your name:",
+      "required": true,
+      "value": "John Doe"
+    },
+    {
+      "type": "email",
+      "name": "email",
+      "label": "E-mail:",
+      "required": true,
+      "value": "john.doe@gmail.com"
+    },
+    {
+      "name": "Chat window title",
+      "type": "title",
+      "label": "Let's talk!",
+    },
+    {
+      "name": "Chat window form info",
+      "type": "information",
+      "label": "Before we start, we'd like to know a few details about you.",
+    }]
+}
+```
+![Filled Form](filled_form.png "filled form example")
+
+To send the data from this form, you can use the **filled form** event. The code sample introduces all currently available field types:
```

Comment:
`field types:` a wymienione są wspólne pola a nie typy

Reply:
description removed from the doc